src/figuresClasses.ts

```diff
@@ -1,11 +1,72 @@
-export interface Figure {}
+export interface Figure {
+  readonly shape: string;
+  readonly color: string;
+  getArea(): number;
+}
```

Comment:
```suggestion
type Color = 'red' | 'green' | 'blue';
type Shape = 'triangle' | 'circle' | 'rectangle';

export interface Figure {
  shape: Shape;
  color: Color;
  getArea(): number;
}
```
specify types more precisely